tools/deploy_firmware.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+
+import time
+import sys
+import os
+import subprocess
+import difflib
+import argparse
+import shutil
+from typing import List
+
+import pexpect
+import pexpect.fdpexpect
+import pexpect.pxssh
+import serial
+
+
+class PrplwrtDevice:
```

Comment:
nitpick: I'd call the base class `Device` and the derived class `prplWrtDevice`, personally.

"prplWrt device" is more specific than "device", so this class hierarchy feels backwards